WordPress/Flags.xcassets/AC.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "AC.png",
```

Comment:
Do we have vector or higher-resolution versions of these assets available?

Reply:
It appears that these images are 72px square, effectively 24pts on larger devices.

Reply:
That's a great question! 

@SylvesterWilmott do you know if we have those somewhere and if so, how could I obtain them?